pybuildkite/client.py

```diff
@@ -72,7 +74,7 @@ def request(
             return response
         if method == "DELETE":
             return response.ok
-        if headers == None or headers.get("Accept") == "application/json":
+        if headers.get("Accept") is None or headers.get("Accept") == "application/json":
```

Comment:
The purpose of not forcing accept headers to be "application/json" is so that we can still respond in text. So we only want to return `response.json()` if `headers.get("Accept") == "application/json"`

Reply:
The behavior that exists currently on master is to return `response.json()` if no header is provided -- this preserves that behavior.

Reply:
I just pushed up a test that asserts this behavior.

Reply:
Oh woops you're right! Thanks for checking me.